content/en/docs/Installation/_index.md

```diff
@@ -355,11 +355,16 @@ The following flags are used to control the behavior of Kyverno and must be set
 The following flags can also be used to control the advanced behavior of Kyverno and must be set on the main `kyverno` container in the form of arguments.
 
 1. `-v`: Sets the verbosity mode of Kyverno log output. Takes an integer from 1 to 6 with 6 being the most verbose.
-2. `--background-scan`: The interval (like 30s, 15m, 12h) for background processing resulting in policy report entries. Default is set to 1h.
-3. `--webhookTimeout`: Sets the timeout for a webhook response from Kyverno. Value is in seconds. This is deprecated in Kyverno 1.5.0.
-4. `--gen-workers`: the number of workers for processing generate policies concurrently. Default is set to 10.
-5. `--generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
-6. `--autoUpdateWebhooks`: auto-configuration of the webhooks based on installed policies. Default is set to "true".
+2. `profile`: setting this flag to 'true' will enable profiling.
+3. `profilePort`: specifies port to enable profiling at, defaults to 6060.
+4. `metricsPort`: specifies the port to expose prometheus metrics, default to port 8000.
+5. `genWorkers`: the number of workers for processing generate policies concurrently. Default is set to 10.
+6. `disableMetrics`: specifies whether (true/false) to enable exposing the metrics. Default is set to "false".
+7. `backgroundScan`: the interval (like 30s, 15m, 12h) for background processing. Default is set to 1h.
+8. `imagePullSecrets`: specifies secret resource names for image registry access credentials.
+9. `autoUpdateWebhooks`: Set this flag to 'false' to disable auto-configuration of the webhook. Default is set to "true".
+10. `generateSuccessEvents`: specifies whether (true/false) to generate success events. Default is set to "false".
```

Comment:
I don't see "generateSuccessEvents" in this code block pointed out by @realshuting 

https://github.com/kyverno/kyverno/blob/e5e849acfebd0461ae147d066821fdf86c620fc7/cmd/kyverno/main.go#L73-L98

Reply:
Is it "deprecated" or is it "removed" in 1.5.0? Docs say "deprecated".

Reply:
Sorry it should be ConfigMap variable:
https://github.com/kyverno/kyverno/blob/e5e849acfebd0461ae147d066821fdf86c620fc7/pkg/config/dynamicconfig.go#L41

Reply:
Where did you say it's deprecated?

Reply:
I have moved `generateSuccessEvents`